walkthroughs/week-5-tdd/intro/src/main/java/com/google/sps/Greeter.java

```diff
@@ -22,6 +22,17 @@
    * Returns a greeting for the given name.
    */
   public String greet(String name) {
-    return "Hello " + name;
+    name = removeNonAlphanumeric(name);
+    return "Hello " + name.trim();
+  }
+
+  /**
+   * Returns non-alphanumeric characters from a string.
+   * @params string   the string to which we want to remove the non-alphanumeric characters.
```

Comment:
Nitpick: is the Javadoc name "@param" (without an "s")?

Reply:
Another nit (optional really): the parameter javadoc isn't really necessary here, as it's clear from the context. You could change 'from a string' to 'from the given string' to make it slightly clearer